app/models/solidus_paypal_commerce_platform/paypal_order.rb

```diff
@@ -61,13 +67,23 @@ def purchase_units(include_shipping_address: true)
     end
 
     def shipping_info
-      {
-        name: {
-          full_name: @order.ship_address.full_name
-        },
-        email_address: @order.email,
-        address: get_address(@order.ship_address)
-      }
+      if greater_than_2_10
+        {
+          name: {
+            full_name: @order.ship_address.name
+          },
+          email_address: @order.email,
+          address: get_address(@order.ship_address)
+        }
+      else
+        {
+          name: {
+            full_name: @order.ship_address.full_name
+          },
+          email_address: @order.email,
+          address: get_address(@order.ship_address)
+        }
+      end
```

Comment:
Any specific reason for duplicating the whole structure instead of just having a dynamic value for `full_name:`?

Reply:
no, I changed it.